docs/env.md

```diff
@@ -1,117 +1,260 @@
 # Environment variables
```

Comment:
Two missing settings:

DJANGO_SENTRY_DSN
Should be set to activate sentry for an environment. The value of this DSN is given when you add a project to your Sentry instance.
- Type: string
- Required: No
- Default: None

DJANGO_ALLOWED_HOSTS
A string of comma separated domains that Django should accept and answer to. See [Django documentation](https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts) for more details about this setting.
- Type: String
- Required: Varies depending on the environment.
  - No in development, this setting is not available and its value is forced to accept any originating domain;
  - Yes otherwise.
- Default: Varies depending on the environment.
  - ["*"] in development: all originating domains are accepted;
  - None otherwise: all originating domains are rejected.